tools/make_patch.c

```diff
@@ -241,7 +240,7 @@ struct symbol *parse_symfile(FILE *file, const char *prefix)
 	}
 
 	if (!symbols) {
-		fprintf(stderr, "Error: No symbols found starting with: %s\n", prefix);
+			fprintf(stderr, "Error: No symbols found.");
```

Comment:
Why the extra indent?

Reply:
Removed extra indent in https://github.com/pret/pokecrystal/pull/882/commits/3a78bcc8073928f2675d66b38124b90b6796ffca